DotNet.Bundle/PlistWriter.cs

```diff
@@ -1,13 +1,16 @@
 using System;
 using System.IO;
 using System.Xml;
-
+using Microsoft.Build.Framework;
+
 namespace Dotnet.Bundle
 {
     public class PlistWriter
     {
         private readonly BundleAppTask _task;
-        private readonly StructureBuilder _builder;
+        private readonly StructureBuilder _builder;
+
+        private const char Separator = ';';

```

Comment:
private const string Separator = ";";

Reply:
The split takes a character, hence the `const char` that  can use in both the Split() and Contains() methods